utils/launch.sh

```diff
@@ -26,8 +26,7 @@ usage() {
 }
 
 NAME="$(basename $0)"
-REAL_NAME="$(readlink -f $0)"
-HERE="$(cd "$(dirname "$REAL_NAME")" && pwd)"
+HERE="$( cd "$( dirname "$( dirname "$0" )" )" && pwd )"
```

Comment:
Ah, you're right. Maybe it would be easiest to do an OS check, and only use `readlink` on non-BSD based systems?